project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module

```diff
@@ -141,26 +141,15 @@ function c4m_features_homepage_voices_views_carousel() {
       'date' => c4m_helper_date_time_ago($wrapper->created->value()),
       'image' => file_create_url($banner['uri']),
```

Comment:
@nirgn975 
> Apply an image style instead of outputting the entire image

This is part of the request, please add an image style for the banner.